source/common/http/mixed_conn_pool.h

```diff
@@ -13,10 +15,13 @@ class HttpConnPoolImplMixed : public HttpConnPoolImplBase {
       Upstream::HostConstSharedPtr host, Upstream::ResourcePriority priority,
       const Network::ConnectionSocket::OptionsSharedPtr& options,
       const Network::TransportSocketOptionsConstSharedPtr& transport_socket_options,
-      Upstream::ClusterConnectivityState& state)
+      Upstream::ClusterConnectivityState& state,
+      absl::optional<HttpServerPropertiesCache::Origin> origin,
```

Comment:
Out of curiosity, when would this not be present?

Reply:
when the flag is off, maybe for H2 over ALTS?